/bin/sh[5]: @mkdir_p@: not found [No such file or directory]

--- w3m-v0.5.6/po/Makefile.in.in.old	2026-02-07 11:33:48.968833499 -0500
+++ w3m-v0.5.6/po/Makefile.in.in	2026-02-07 11:34:36.909343192 -0500
@@ -42,7 +42,7 @@
 mkinstalldirs = $(SHELL) @install_sh@ -d
 install_sh = $(SHELL) @install_sh@
 MKDIR_P = @MKDIR_P@
-mkdir_p = @mkdir_p@
+mkdir_p = @MKDIR_P@
 
 # When building gettext-tools, we prefer to use the built programs
 # rather than installed programs.  However, we can't do that when we
